core/templates/dev/head/services/ContextService.js

```diff
@@ -110,6 +140,12 @@ oppia.factory('ExplorationContextService', [
       isInExplorationEditorMode: function() {
         return (this.getPageContext() === PAGE_CONTEXT.EDITOR &&
             this.getEditorTabContext() === EDITOR_TAB_CONTEXT.EDITOR);
+      },
+
+      // Following variable helps to know whether question editor is
```

Comment:
Docstring isn't clear, doesn't this just tell us whether we are in the question editor mode? Never copy docstrings blindly.

Reply:
I don't think we need this function for now as there's only question_editor page for now